indra/newview/llspatialpartition.h

```diff
@@ -205,6 +205,7 @@ class LLDrawInfo : public LLRefCount
 LL_ALIGN_PREFIX(64)
 class LLSpatialGroup : public LLOcclusionCullingGroup
 {
+	using super = LLOcclusionCullingGroup;
```

Comment:
What is this kind of Java-style keyword thing?

Reply:
This is a common way we widely use to fill the hole in C++ standard :-)
![image](https://github.com/secondlife/viewer/assets/124201357/d15e1cac-dab1-40fd-980c-c31af9e41edd)